src/page/Auth/Signup/component/CustomButton/index.tsx

```diff
@@ -2,18 +2,19 @@ import styles from './CustomButton.module.scss';
 
 interface ButtonProps {
   content:string,
-  buttonType:string,
+  buttonSize:string,
```

Comment:
```suggestion
  buttonSize: 'small' | 'mobile' | 'large',
```
클래스에 맞게 작성해두는 게 좋아보이죠?!